src/extractors/custom/www.newyorker.com/index.js

```diff
@@ -4,15 +4,15 @@
 export const NewYorkerExtractor = {
   domain: 'www.newyorker.com',
   title: {
-    selectors: ['h1.title'],
+    selectors: ['h1[class^="ArticleHeader__hed"]'],
   },
 
   author: {
-    selectors: ['.contributors'],
+    selectors: ['div[class^="ArticleContributors"] a[rel="author"]'],
   },
 
   content: {
-    selectors: ['div#articleBody', 'div.articleBody'],
+    selectors: ['main[class^="Layout__content"]'],
```

Comment:
The new selector includes an additional section that is not part of the core content block (`footer.ArticleFooter__footer___3-wlJ`), could you please clean that out?